mobile/tsconfig.json

```
@@ -1,6 +1,10 @@
 {
   "extends": "expo/tsconfig.base",
   "compilerOptions": {
-    "strict": true
+    "strict": true,
+    "baseUrl": ".",
+    "paths": {
+      "*": ["src/*"]
+    }
```

`baseUrl` this will suffice if we use `src/` in imports
```suggestion
    "baseUrl": "."
```